controllers/slicegateway/slicegateway.go

```diff
@@ -960,10 +996,36 @@ func getAddrSlice(nodeIPS []string) []corev1.EndpointAddress {
 	}
 	return endpointSlice
 }
-func validateEndpointAddresses(subset corev1.EndpointSubset, remoteNodeIPS []string) bool {
+func nodeIPsCompletelyDifferent(subset corev1.EndpointSubset, remoteNodeIPs []string) bool {
```

Comment:
@YachikaRalhan now both method validateNodeIPsInEndpoint and nodeIPsCompletelyDifferent seems to do the same thing right?

Reply:
No please check this example to more details, the if condition is different - https://go.dev/play/p/UZEqnK_GtoU